### ISSUE TITLE
navsounds is making Google Chrome inaccessible to NVDA

### ISSUE THREAD

**Issue by edu-mx** [ahmedthebest31/navsounds#4]
Hello, I wanted to say that I am having problems accessing pages in my Google Chrome web browser because of NavSounds.
I'm sure of this because I've already tested it on 2 computers and both happen the same thing.
The problem is that nvda stops speaking the content of any page.
I believe this is happening because navsound only speaks the items it can play, and possibly some things in the browser were not programmed and it simply remains silent.

**Comment by ahmedthebest31** [comment]
Hello,

I want to address the issue you're experiencing with NavSound in Google Chrome. I never had this issue on my main computer, and the other two laptops I tried it with also didn't exhibit this problem. This is the first time someone has reported this specific issue to me.

https://github.com/ahmedthebest31/navsounds/assets/30302865/7d3d731b-3ce5-482c-99de-a8a033fd2fd5


I am currently using NavSound with the latest version of Google Chrome and NVDA version 2023.2 on Windows 11. To help me better understand and address the issue, could you please provide information about the specific version of the NavSound addon you are using? Is it version 1.4 or an earlier version?

I have also included a video to demonstrate how I am using it right now. If you could provide a video demonstrating the problem, it would be immensely helpful for me to identify and resolve the issue.

Thank you for bringing this problem to my attention, and I'm committed to doing my best to fix it as soon as possible.

**Comment by edu-mx** [comment]
Hello, sorry for the lack of details.
Here is some useful information and attached is an audio demo;
NVDA version 2023.2, windows 10 home and google chrome in the latest version.
Today in a new test I realized that now nvda is no longer like before, it only said "unknown, unknown" for all elements on the web screen.
It's normal, but I still face the problem of great slowness when browsing the demonstrated website "python.org".
Note that in the demo, at times everything is silent (At that moment I was pressing the up and down arrows to navigate.
Then, so you could know which keys I'm pressing, I activated the "insert + 4" mode and noticed that the slowness continues to be great.
     I believe that this also has a consequence because of my computer, which i
[demonstration.zip](https://github.com/ahmedthebest31/navsounds/files/12885403/demonstration.zip)
s not good at all (amd C50 processor, 200 gb of space and 4 gb of ram).
When I disable navsounds, navigation is normal, without any browser crashes.

**Comment by ahmedthebest31** [comment]
I just released a new version finally I abled to fix this issue it should work perfectly right now

**Comment by edu-mx** [comment]
@ahmedthebest31 thank you for taking the time to resolve this issue, I'm very happy. The problem was successfully resolved.
